### ISSUE TITLE
[Feature] Split Tunnel with regex or mask...

### ISSUE THREAD

**Issue by HamidZaeri** [Windscribe/Desktop-App#69]
Hi

Ability to add hostname like "*google.com" to Split tunneling, so all subdomain of google.com would be excluded.

And Even further, possibility to add "*.us" or "*.de" so we can exclude all domains specified to our country because usually those domains are not restricted.

Thanks

**Comment by d7an** [comment]
> Ability to add hostname like "*google.com" to Split tunneling, so all subdomain of google.com would be excluded.
> 
> And Even further, possibility to add "_.us" or "_.de" so we can exclude all domains specified to our country because usually those domains are not restricted.

Strongly agree this this feature request. Especially on macOS version where there're a limit of 50ish domain that can be added to the split tunnel exclude list.

**Comment by d7an** [comment]
Better yet, support specifying the split tunnel host/ip using an external file such as an autoproxy PAC file format.

**Comment by sallymanda** [comment]
Agreed. Managing multiple domains would be especially useful for cases like `archive.md`.
